src/node/_stream_duplex.js

```diff
@@ -3,16 +3,6 @@
 //     https://opensource.org/licenses/Apache-2.0
 //
 /* eslint-disable */
-import {
-  Duplex,
-  from,
-  fromWeb,
-  toWeb,
-} from 'node-internal:streams_duplex';
-export {
-  Duplex,
-  from,
-  fromWeb,
-  toWeb,
-};
+import { Duplex, from, fromWeb, toWeb } from "node-internal:streams_duplex";
+export { Duplex, from, fromWeb, toWeb };
```

Comment:
Here too, I much prefer the original formatting for these.

Reply:
For now does prettier have an option to ignore lines/blocks using comments? If so, let's use that with these

Reply:
`// prettier-ignore`

Added